scripts/push_and_create_pr.sh

```diff
@@ -1,6 +1,11 @@
 #! /usr/bin/env bash
 set -x
-AUTO_APPROVE_LABEL="auto-approve"
+
+if [ $BUILD_REASON == "Manual" ]; then
+    APPROVE_LABEL="manual-approve"
+else
+    APPROVE_LABEL="auto-approve"
+fi
```

Comment:
```suggestion
if [ $BUILD_REASON == "Manual" ]; then
    APPROVE_LABEL="Automated PR"
else
    APPROVE_LABEL="auto-approve"
fi
```

What do you think about renaming the label here in the other case? Manual-approve seems a little odd for the general case -- approved otherwise

Reply:
True. Although it's not always the case (The manual build reason is literally ... Manual) Perhaps we could just rename it to `manual-trigger` `Manually triggered` or something like this ? It's not going to set the label.

Now that I'm thinking about this we might only set that variable if it's not Manual run